charts/grafana/templates/_helpers.yaml

```
@@ -46,8 +46,9 @@ grafana.{{ .Values.global.baseDomain }}
 {{ default (printf "%s-grafana-backend" .Release.Name) .Values.backendSecretName }}
 {{- end }}
 
+
 {{ define "grafana.bootstrapper.serviceaccount" -}}
-{{ printf "%s-grafana-bootstrapper" .Release.Name }}
+  {{ printf "%s-grafana-bootstrapper" .Release.Name }}
```

This leading space is inconsistent with similar templates.
```suggestion
{{ printf "%s-grafana-bootstrapper" .Release.Name }}
```